build.py

```diff
@@ -208,99 +163,98 @@ def build_variable_fonts(designspace, *steps):
         )
 
     if args.static_fonts == True:
+        # Prepare masters for interpolation.
+        generator = fontmake.instantiator.Instantiator.from_designspace(designspace)
+
         for instance_descriptor in designspace.instances:
+            # Generate instances once.
+            instance = generator.generate_instance(instance_descriptor)
+            glyphsLib.interpolation.apply_instance_data_to_ufo(instance, instance_descriptor, designspace)
 
             build_font_instance(
-                generator, 
-                instance_descriptor,
+                copy.deepcopy(instance),
                 step_set_feature_file(INPUT_DIR / "features" / "features_code.fea"),
             )
 
             if not args.no_mono:
                 build_font_instance(
-                    generator,
-                    instance_descriptor,
+                    copy.deepcopy(instance),
                     step_set_font_name("Cascadia Mono"),
                     step_set_feature_file(INPUT_DIR / "features" / "features_mono.fea"),
                 )
 
             if not args.no_powerline:
                 build_font_instance(
-                    generator,
-                    instance_descriptor,
+                    copy.deepcopy(instance),
                     step_set_font_name("Cascadia Code PL"),
                     step_set_feature_file(INPUT_DIR / "features" / "features_code_PL.fea"),
                     step_merge_pl,
                 )
 
                 if not args.no_mono:
                     build_font_instance(
-                        generator,
-                        instance_descriptor,
+                        copy.deepcopy(instance),
                         step_set_font_name("Cascadia Mono PL"),
                         step_set_feature_file(INPUT_DIR / "features" / "features_mono_PL.fea"),
                         step_merge_pl,
                     )
 
             if not args.no_nerdfonts:
                 build_font_instance(
-                    generator,
-                    instance_descriptor,
+                    copy.deepcopy(instance),
                     step_set_font_name("Cascadia Code NF"),
                     step_merge_nf,
                 )
 
                 if not args.no_mono:
                     build_font_instance(
-                        generator,
-                        instance_descriptor,
+                        copy.deepcopy(instance),
                         step_set_font_name("Cascadia Mono NF"),
-                        step_remove_ligatures,
                         step_merge_nf,
```

Comment:
I think `step_remove_ligatures` was replaced with `step_set_feature_file`. The `Mono` NF version does still need one of those two things, but I'm no longer sure which.

Reply:
As Dustin mentions, this should be `step_set_feature_file(INPUT_DIR / "features" / "features_mono_PL.fea"),`. I've updated it in my build script, but haven't pushed it out yet :). 

The NF variant would use the same feature file as the PL variant, so the code can by copied directly.

Reply:
I was indirectly wondering if "Cascadia Mono NF" was still a thing, since the build script would have blown up upon encountering that unknown symbol.

Reply:
Ah. At the moment it isn't being built, no. But I'm hopeful we can get it re-instated.